capture/src/main/res/color/capture_mode_text_color.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:color="?android:attr/textColorPrimary" android:state_activated="true"/>
+    <item android:color="?android:attr/textColorPrimary" android:state_enabled="true"/>
```

Comment:
This is not required, the control assumes selection is done via the `active` state.

Reply:
selection is done but text seems to be disabled there

Reply:
that just means that there's a bug in the state management and this is an incorrect fix.

Reply:
Done